ple/games/tetris.py

```diff
@@ -0,0 +1,71 @@
+import base
+import pygame
+import numpy as np
+import tetris_engine as game_engine
+
+WINDOWWIDTH = 200
+WINDOWHEIGHT = 400
```

Comment:
set this as default values in the `__init__` function signature for width and height instead of a constant outside of the class.